scio-avro/src/main/scala/com/spotify/scio/coders/AvroKryoRegistrar.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.spotify.scio.coders
+
+import com.spotify.scio.coders.instances.kryo.{GenericAvroSerializer, SpecificAvroSerializer}
+import com.twitter.chill._
+import org.apache.avro.generic.GenericRecord
+import org.apache.avro.specific.SpecificRecordBase
+
+@KryoRegistrar
+class AvroKryoRegistrar extends IKryoRegistrar {
+  override def apply(k: Kryo): Unit = {
+    k.forSubclass[SpecificRecordBase](new SpecificAvroSerializer)
```

Comment:
Not 100% sure but I think is is used to handle classes containing avro fields.
When serialized with kryo, we want to use the avro serialization for those fields instead of the default kryo one.